dev-packages/e2e-tests/test-applications/solidjs/package.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "solidjs",
+  "version": "0.0.0",
+  "description": "",
+  "scripts": {
+    "build": "vite build",
+    "clean": "npx rimraf node_modules pnpm-lock.yaml dist",
+    "dev": "vite",
+    "preview": "vite preview",
+    "start": "vite",
+    "test:prod": "TEST_ENV=production playwright test",
+    "test:build": "pnpm install && npx playwright install && pnpm build",
+    "test:assert": "pnpm test:prod"
+  },
+  "license": "MIT",
+  "devDependencies": {
+    "@playwright/test": "^1.43.1",
```

```suggestion
    "@playwright/test": "^1.44.1",
```

let's use latest here!

---

ah, and also add `@sentry-internal/test-utils` as a dep, explicitly.